projects/lcms/cms_transform_extended_fuzzer.c

```diff
@@ -132,6 +144,10 @@ run_test(const uint8_t *data,
       cmsCIELab Lab1;
       cmsDoTransform(hTransform, input, &Lab1, 1);
     }
+    else if (dstFormat == (FLOAT_SH(1)|COLORSPACE_SH(PT_MCH3)|CHANNELS_SH(3)|BYTES_SH(0))){
+      cmsCIELab okLab;
+      cmsDoTransform(hTransform, input, &okLab, 1);
```

Comment:
If I remember corretly, the relationship between the `dstProfile` and `cmsDoTransform` is quite close, could you just give a small comment on why this one is correct? Is it inspired by a testcase?

Reply:
We found type for OkLab profile defined in Little-CMS/testbed/testcms2.c, which is **(FLOAT_SH(1)|COLORSPACE_SH(PT_MCH3)|CHANNELS_SH(3)|BYTES_SH(0))** and OkLab is a Lab profile, so we used  **cmsCIELab** for the output buffer. According to the Little-CMS testcases, it should be correct. We noticed that **else if** at the 143 and 147 lines have the same logic, so we will merge them.